pkg/cmd/extension/command_test.go

```diff
@@ -27,7 +28,9 @@ import (
 func TestNewCmdExtension(t *testing.T) {
 	tempDir := t.TempDir()
 	oldWd, _ := os.Getwd()
-	assert.NoError(t, os.Chdir(tempDir))
+	localExtensionTempDir := filepath.Join(tempDir, "gh-hello")
+	assert.NoError(t, os.MkdirAll(localExtensionTempDir, 0755))
+	assert.NoError(t, os.Chdir(localExtensionTempDir))
```

Comment:
Had to do this because these tests started failing with the name validation was added.